### PR TITLE
Feat: select operator

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

File: app/pages/_app.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { getClientEnvironment } from "../lib/relay/client";
 import "normalize.css";
 import BCGovTypography from "components/BCGovTypography";
 import SessionExpiryHandler from "components/SessionExpiryHandler";
+import "semantic-ui-css/semantic.min.css";
```
Comment (matthieu-foucault) [inline app/pages/_app.tsx:8]: that's a lot of CSS to import in every page for just one component

Comment (matthieu-foucault) [reply app/pages/_app.tsx:8]: Something like MUI's [Autocomplete](https://mui.com/components/autocomplete/) might be a better option if we want to bring in a single, customizable, component

Comment (pbastia) [reply app/pages/_app.tsx:8]: We might get away with just a few files: https://github.com/Semantic-Org/Semantic-UI-React/issues/2029#issuecomment-326455590

Comment (matthieu-foucault) [reply app/pages/_app.tsx:8]: I'm not super familiar with SemanticUI, but it looks much easier to make a theme for MUI than SemanticUI.
Also there's a [useAutocomplete hook](https://mui.com/components/autocomplete/#useautocomplete) if we want to not import any styles

Comment (dleard) [reply app/pages/_app.tsx:8]: I'll give MUI Autocomplete a look. I figured importing SemanticUI's CSS would just be a temporary solution so I could continue moving forward on developing this feature for now

Comment (matthieu-foucault) [reply app/pages/_app.tsx:8]: You can also start with a simple select, and we add a searchable one after

File: app/package.json [hunk 2]
```diff
@@ -60,10 +60,14 @@
     "@bcgov-cas/sso-react": "^1.0.1",
     "@bcgov/bc-sans": "^1.0.1",
     "@button-inc/bcgov-theme": "^1.0.1",
+    "@emotion/react": "^11.7.0",
+    "@emotion/styled": "^11.6.0",
```
Comment (matthieu-foucault) [inline app/package.json:64]: where is this used? We already have two different styling libraries (`styled-jsx` and `styled-components`)

File: app/lib/theme/widgets/DisplayOnlyWidget.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const DisplayOnlyWidget: React.FC<WidgetProps> = ({ options }) => {
+  return (
+    <span className="paragraph-text">
+      <label>{options.title}</label>
+      <p>{options.text}</p>
+      <style jsx>{`
+        h3 {
+          font-size: 1.25rem;
+          margin-bottom: 0.5rem;
+        }
```
[the remark this answers is not in the document]
Comment (dleard) [reply app/lib/theme/widgets/DisplayOnlyWidget.tsx:12]: I'll remove it, copy paste from CIIP

File: app/package.json (hunk 2 above)
Comment (dleard) [reply app/package.json:64]: They came with @mui I can remove them and see if it affects the Autocomplete

Comment (dleard) [reply app/package.json:64]: mui Autocomplete explodes without those dependencies

Comment (matthieu-foucault) [reply app/package.json:64]: I see emotion can be replaced by styled-components, but it's currently discouraged as there is an issue with styled-components, SSR and MUI: https://mui.com/guides/styled-engine/#how-to-switch-to-styled-components

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]

**Review by matthieu-foucault — APPROVED** [review]

**Pull request merged**